protocol-specs/generators/multisignature/README.md

```diff
@@ -0,0 +1,20 @@
+# Multisignature Transaction Signing and Serialisation
+
+A set of test generators for updated multisignatures transactions by LIP-0017
+
+### Format
+
+#### Input
+
+- account: sender account
+- transaction: raw transaction
+- network identifier: network identifier used
+- coSigner: accounts that co-signed the transaction
```

Comment:
It's not really important, but in some test cases not all entries of `coSigner` are used. Maybe you want to delete them from the JSON?

Reply:
I updated the coSigner to be the correct ones.